<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>397</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>459</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Usb Creator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>To try or install Ubuntu from a USB disk, it needs to be set up as a startup disk.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Source disk image (.iso) or CD:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTreeWidget" name="ui_source_list">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Image</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>OS Version</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Please insert a CD or select 'Other...'.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="ui_add_source">
       <property name="text">
        <string>Other...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>USB Disk to use:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QTreeWidget" name="ui_dest_list">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Device</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Label</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Capacity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Free Space</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="ui_format_dest">
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QFrame" name="ui_persist_frame">
     <layout class="QVBoxLayout" name="ui_persist_vbox" stretch="0,0,0,0">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>When starting up from this disk, documents and settings will be:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ui_persist_enabled">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Stored in reserved extra space</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ui_persist_hbox">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>25</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="ui_persist_text">
          <property name="text">
           <string>How Much</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="ui_persist_slider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ui_persist_label">
          <property name="text">
           <string>0.0 MB</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="ui_persist_disabled">
        <property name="text">
         <string>Discarded on shutdown, unless you save them elsewhere</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="ui_start_button">
       <property name="text">
        <string>Make Startup Disk</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="ui_quit_button">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ui_persist_disabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>ui_persist_slider</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_persist_enabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>ui_persist_slider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_persist_enabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>ui_persist_text</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_persist_enabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>ui_persist_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_persist_disabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>ui_persist_text</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_persist_disabled</sender>
   <signal>clicked(bool)</signal>
   <receiver>ui_persist_label</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_quit_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
